.github/actions/update-tool-build/action.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Update Tool build'
+description: 'Update Tool build'
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout Code
+      uses: actions/checkout@v3
+
+    # - name: print g++ version
+    #   shell: bash
+    #   if: startsWith(matrix.os,'ubuntu') || startsWith(runner.os,'ubuntu') ||  startsWith(matrix.os,'Windows') || startsWith(runner.os,'Windows')
+    #   run: |
+    #     if [ '$runner_os' == 'Linux' || '$matrix_os' == 'Linux' || '$runner_os' == 'Windows' || '$matrix_os' == 'Windows' ]; then
+    #       gcc --version
+    #     elif [ '$runner_os' == 'macOS' || '$matrix_os' == 'macOS' ]; then
+    #       brew info gcc
+    #     fi
+
+    - name: Set up GCC for Linux
+      if: startsWith(matrix.os,'ubuntu') || startsWith(runner.os,'ubuntu')
+      uses: egor-tensin/setup-gcc@v1
+      with:
+        version: latest
+        platform: x64
+
+    # - name: Set up MinGW for Windows
```

Comment:
Is there a reason why these are currently commented out?

Reply:
Latest windows runners now come with sufficient MinGW version and as the Windows runners are very slow I removed the download and install of latest MinGW for performance improvements.
Should I remove the code?